workspace-loader/pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2018-2018 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-parent</artifactId>
+        <groupId>org.eclipse.che</groupId>
+        <version>6.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <groupId>org.eclipse.che.workspace.loader</groupId>
+    <artifactId>che-workspace-loader</artifactId>
+    <version>6.2.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>Che Workspace Loader :: Web App</name>
+    <inceptionYear>2018</inceptionYear>
+    <dependencies>
+        <dependency>
```

Comment:
could you please explain why we're having this dependency ?
AFAIK this module is not using it.

Reply:
Removed as unused. Thanks.